pixano/utils/python.py

```diff
@@ -123,3 +123,10 @@ def fn_sort_dict(dict_: dict[str, Any], order_by: list[str], descending: list[bo
         else:
             key.append(value)
     return tuple(key)
+
+
+def uniq_list(seq: list) -> list:
```

Comment:
Rename seq to list_ if it only deal with lists or sequence if it deals with sequences

In the second case, change the type to Sequence.

Reply:
done